### PR TITLE
Origami add borrowed metric

### PR DESCRIPTION
**Pull request by frost-ostrich**
##### Name (to be shown on DefiLlama): 
Origami

##### Twitter Link: 
https://x.com/origami_fi

##### List of audit links if any: 
https://docs.origami.finance/technical-reference/audits

##### Website Link: 
https://origami.finance/

##### Logo (High resolution, will be shown with rounded borders): 
image

##### Current TVL: 
9.82 M

##### Chain: 
Ethereum, Arbitrum & soon un Bera Chain

##### Short Description (to be shown on DefiLlama): 
Origami is a first-in-class automated leverage protocol launching on Ethereum and 🐻

##### Category (full list at https://defillama.com/categories) *Please choose only one: 
Leveraged Farming

##### Oracle Provider(s): Specify the oracle(s) used (e.g., Chainlink, Band, API3, TWAP, etc.): 
Chainlink

##### Documentation/Proof: Provide links to documentation or any other resources that verify the oracle's usage:
https://docs.origami.finance/technical-reference/contracts#oracles

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/origami exports TVL: 
        
 
 
    ethereum-borrowed         39.13 M
    borrowed                  39.13 M
    ethereum                  8.46 M
    arbitrum                  97.12 k
    arbitrum-borrowed         0
    
    total                    8.56 M

File: projects/origami/index.js [hunk 1]
```diff
@@ -13,18 +13,51 @@ module.exports = {
...
+  endpoint,
+) {
+  const { metrics: [{  investmentVaults }] } = await cachedGraphQuery('origami/' + chain, endpoint, '{ metrics { investmentVaults { id kinds } } }')
+  await Promise.all(
+    investmentVaults.map(async vault => {
+      const isLeveraged = !!vault.kinds.find( v => v === 'Leverage')
+      let token = await api.call({ abi: 'address:reserveToken', target: vault.id })
+
+      if (isLeveraged) {
+        const assetsAndLiabilities = await api.call({ abi: 'function assetsAndLiabilities() external view returns (uint256 assets,uint256 liabilities,uint256 ratio)', target: vault.id })
```
Comment (frost-ostrich) [inline projects/origami/index.js:33]: These rpc calls are being done twice in both `tvl` and `liabilities` functions, would it be possible to make the rpc calls before hand and pass the results as params of each function? tried couple of different things but couldn't make it work

**Comment by llamatester** [comment]
The adapter at projects/origami exports TVL: 
        
 
 
    ethereum                  7.50 M
    arbitrum                  97.65 k
    
    total                    7.60 M

File: projects/origami/index.js (hunk 1 above)
Comment (g1nt0ki) [reply projects/origami/index.js]: hi @frost-ostrich thanks for the update, I refactored it to batch query. Also, removed the borrowed field as we use it only for lending protocols not leverage protocols (atm)

Comment (frontier159) [reply projects/origami/index.js]: @g1nt0ki origami is similar to Contango in that regard, which does show borrowed 

https://defillama.com/protocol/contango-v2#information

Could you please advise why it does show the debt but origami cannot? 

Thank you!

Comment (frontier159) [reply projects/origami/index.js]: @g1nt0ki could you please check the above comment when you get time?

**Pull request merged**